docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       - ./scripts/docker/env/production/django.env
 
   geoserver:
-    image: geonode/geoserver
+    image: waybarrios/geoserver:piensa
```

Comment:
Good idea @francbartoli! I just did it [here](https://github.com/GeoNode/geonode/pull/2855/commits/950af68c985b2b33fc7d3eaae1c8fc52913fda62)